### DISCUSSION TITLE
Q: 变量名self.url抓取的文章地址多了个斜杠 || Q: There is an extra slash in the article address captured by the variable name self.url

### DISCUSSION THREAD

**Discussion by Motalyss** [walinejs/waline#2425] [Q&A] [answered]
#### Describe the bug | 问题描述

评论的原文章地址为：https://motalyss.github.io/posts/kimihadouikiruka

但自定义回复邮件内容里使用 self.url 或者 site.postUrl 抓取的地址为：https://motalyss.github.io//posts/kimihadouikiruka#66179fd4ec2ef817e546925f
posts前有两条斜杠，点进这个链接就404了。。。

不知道是不是哪里设置得有问题

### Website URL | 问题网站

https://motalyss.github.io//posts/kimihadouikiruka#66179fd4ec2ef817e546925f

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
#### Describe the bug | Problem description

The original article address of the comment is: https://motalyss.github.io/posts/kimihadouikiruka

But the address captured using self.url or site.postUrl in the custom reply email content is: https://motalyss.github.io//posts/kimihadouikiruka#66179fd4ec2ef817e546925f
There are two slashes before posts, and clicking on this link will result in a 404. . .

I don’t know if there is something wrong with the settings.

### Website URL | Problem website

https://motalyss.github.io//posts/kimihadouikiruka#66179fd4ec2ef817e546925f

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
SITE_URL 环境变量最后不要加 `/` 即可。

---
Just don’t add `/` at the end of the SITE_URL environment variable.
